app/models/vm_or_template/operations/configuration.rb

```diff
@@ -74,8 +74,10 @@ def disconnect_floppies
   def raw_add_disk(disk_name, disk_size_mb, options = {})
     raise _("VM has no EMS, unable to add disk") unless ext_management_system
     if options[:datastore]
-      datastore = Storage.find_by(:name => options[:datastore])
-      raise _("Data Store does not exist, unable to add disk") unless datastore
+      datastore = ext_management_system.hosts.detect do |h|
+        h.writable_storages.find_by(:name => options[:datastore])
+      end.uniq.compact.first
```

Comment:
@gmcculloug ofcourse - i thought i squashed a local change to the commit which did exactly that. The fix should have been similar to https://github.com/ManageIQ/manageiq-providers-ovirt/pull/42/files#diff-17198062ae8d3e8788a4f5db1eb17ccdR59 - I'll add a test as well.